e2e/cloud_manager/cloud_manager_alerts_test.go

```diff
@@ -0,0 +1,170 @@
+// Copyright 2020 MongoDB Inc
```

Comment:
nit on the naming of the file (and this is also true for atlas) you're already in the cloud_manager folder, let's please rename the file to just `alerts_test.go`